collect_app/src/main/java/org/odk/collect/android/preferences/AboutPreferencesFragment.java

```diff
@@ -69,32 +68,23 @@ public void onStart() {
     }
 
     @Override
+    public void onDestroy() {
+        getActivity().unbindService(websiteTabHelper.getServiceConnection());
+        getActivity().unbindService(forumTabHelper.getServiceConnection());
+        super.onDestroy();
+    }
+
+    @Override
     public boolean onPreferenceClick(Preference preference) {
         final String APP_PACKAGE_NAME = getActivity().getPackageName();
 
         switch (preference.getKey()) {
             case KEY_ODK_WEBSITE:
-                if (websiteTabHelper.getPackageName(getActivity()).size() != 0) {
-                    CustomTabsIntent customTabsIntent =
-                            new CustomTabsIntent.Builder()
-                                    .build();
-                    customTabsIntent.intent.setPackage(websiteTabHelper.getPackageName(getActivity()).get(0));
-                    customTabsIntent.launchUrl(getActivity(), websiteUri);
-                } else {
-                    startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(ODK_WEBSITE)));
-                }
+                websiteTabHelper.openUri(getActivity(), websiteUri);
```

Comment:
Great job cleaning this up, this is much nicer.

Reply:
😄